multicluster/service-mirror/cluster_watcher.go

```diff
@@ -196,6 +197,7 @@ func NewRemoteClusterServiceWatcher(
 		repairPeriod:            repairPeriod,
 		liveness:                liveness,
 		headlessServicesEnabled: enableHeadlessSvc,
+		gatewayAlive:            gatewayAlive,
```

Comment:
`NewRemoteClusterServiceWatcher` is always called with `gatewayAlive == true`. Therefore, lets just remove the `gatewayAlive` argument from this function and pass in `true` here. It should result in a smaller change.

Reply:
Sure, if we actually don't need to instantiate it with `gatewayAlive==false` in all cases. And a new commit was submitted.